clang/include/clang/Basic/Attr.td

```diff
@@ -1135,6 +1135,28 @@ def SYCLKernel : InheritableAttr {
   let Documentation = [SYCLKernelDocs];
 }
 
+// Marks functions which must not be vectorized via horizontal SIMT widening,
+// e.g. because the function is already vectorized. Used to mark SYCL
+// explicit SIMD kernels and functions.
+def SYCLSimd : InheritableAttr {
```

Comment:
Wow, I have seen this attribute in the previous patch.
Should It really be here?

Reply:
right, it should not.